main.go

```diff
@@ -161,6 +161,8 @@ func newGrpcProxy(cfg *config.Config, tlscfg *tls.Config) []grpc.ServerOption {
 		grpc.UnknownServiceHandler(handler),
 		grpc.StreamInterceptor(proxyInterceptor.Stream),
 		grpc.StatsHandler(statsHandler),
```

Comment:
It would be great to have these values be configurable.  Could you add these to config/config.go in the Config struct, maybe as GRPCMaxRxMsgSize and GRPCMaxTxSize , and then setting sane default values in config/default.go ?  Perhaps setting the current default of 4MB is still sane here, 1gb seems like it might be excessive for many people?  And if it's configurable, this allows people to relax the limits.